themes/timer-hugo/layouts/partials/header.html

```diff
@@ -4,7 +4,7 @@
             <div class="col-lg-12">
                 <nav class="navbar navbar-expand-lg navbar-light bg-light">
                     <a class="navbar-brand" href="{{ .Site.BaseURL }}">
-                        <img src="{{ .Site.Params.logo | absURL }}" height="50em" width="auto" alt="logo">
+                        <img src="{{ .Site.Params.logo | absURL }}" height="100em" width="auto" alt="logo">
```

Comment:
I meant this: https://github.com/insilichem/insilichem_com/pull/6/files#r425237591